res/css/views/settings/_Notifications.scss

```diff
@@ -68,6 +68,8 @@ limitations under the License.
 
 .mx_UserNotifSettings_notifTable .mx_Spinner {
     position: absolute;
+    left: 0;
```

Comment:
Rather than repairing the position like this, I think it would be better to fix the spinner's positioned parent. If you set `position: relative;` on `.mx_UserNotifSettings_notifTable`, this causes the spinner to be positioned relative to the notif table, which appears to give a reasonable result.